openshift/jenkins-persistent-template.json

```diff
@@ -281,13 +281,13 @@
       "name": "JENKINS_IMAGE",
       "displayName": "Jenkins docker image reference",
       "description": "Full name of jenkins image for example ",
-      "value": "aerogear/jenkins-2-centos7"
+      "value": "openshift3/jenkins-2-rhel7"
```

Comment:
This image requires setting up Red Hat docker registry and you most likely need a subscription as well. Lets use public image from docker hub - docker.io/openshift/jenkins-2-centos7